src/javaRosa/util.js

```diff
@@ -14,7 +14,7 @@ define([
     util
 ) {
     var SUPPORTED_MEDIA_TYPES = ['image', 'audio', 'video', 'video-inline', 'expanded-audio'],
-        ITEXT_TYPES = _.union(['default'], SUPPORTED_MEDIA_TYPES),
+        ITEXT_TYPES = ['default', 'audio', 'image', 'video', 'video-inline', 'expanded-audio'],
```

Comment:
The ordering mattered for some tests. I think I could have just changed what the export expected, but didn't think too hard about it. At least it's one less hard coded array